env/prod/env/android/main.cljs

```diff
@@ -1,11 +0,0 @@
-(ns env.android.main
-  (:require [status-im.android.core :as core]
-            [re-frame.interop :as interop]
-            [reagent.impl.batching :as batching]))
-
-(set! interop/next-tick js/setTimeout)
```

Comment:
where was this moved? just to make sure we haven't lost it

Reply:
yeah lost in merge

Reply:
I put it back, it's only for prod?

Reply:
no for all builds, you can put it in core